arm-compute/compositeComputeClient.json

```diff
@@ -4,7 +4,7 @@
         "description": "Composite Swagger for Compute Client"
     },
   "documents": [
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/2015-06-15/swagger/compute.json",
+    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/2016-03-30/swagger/compute.json",
     "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/2015-11-01-preview/swagger/containerService.json"
```

Comment:
Please don't replace the old swagger spec with the new version. We need it for getting history of what changed over time. Add a new folder for the api-version and a new swagger spec over there. Take a look at this: https://github.com/Azure/azure-rest-api-specs/tree/master/arm-storage